supabase/README.md

```diff
@@ -0,0 +1,67 @@
+# How to use Supabase Dev Environment
+
+For full Supabase-provided documentation, go [here](https://supabase.com/docs/guides/cli/local-development)
+
+## Prerequisites
+
+- [Install Docker](https://docs.docker.com/get-docker/)
+- [Install Supabase Command Line Interface (CLI)](https://github.com/supabase/cli)
+
+## Running the Dev Environment
+1. Start Docker
+2. After ensuring that docker is running, type `supabase start` from the command line in the root directory, which uses Docker to start supabase services (this command may take a while if it’s your first time using the CLI)
+3. This will generate a series of urls and keys. For basic development roles, note the values for “API URL” and “service_role key” and record them in a .env file which you create and place in the root directory of the repository, in the following format:
+```
+SUPABASE_API_URL=YOUR_API_URL
+SUPABASE_API_KEY=YOUR_SERVICE_ROLE_KEY
+```
+4. You can now access the Local Supabase Dashboard at http://localhost:54323/ (unless configured otherwise)
+5. In the Supabase Dashboard, copy and paste the migration script `primary_migration.sql` into the SQL Editor at address http://localhost:54323/project/default/sql/new
+6. To stop the dev environment, type `supabase stop`
```

Comment:
Nit: you can have a step before this which says "you are now set up" to make it very clear that you are good to go